codex-rs/execpolicy2/src/default.codexpolicy

```diff
@@ -0,0 +1,86 @@
+prefix_rule(
+    pattern = ["git", "status"],
```

Comment:
There are some weird cases where `git status` could have side-effects (ask chat), so we shouldn't put it in the default policy.

Again, I'm not convinced we should be declaring this as part of execpolicy itself.